### ISSUE TITLE
adduct definitions in ION_TYPE

### ISSUE THREAD

**Issue by michaelwitting** [MassBank/MassBank-web#217]
Hi,

I have seen that there are only limited number of adducts supported in the ION_TYPE field.
I'm currently collecting adduct definitions and synonyms in different software tools, DBs etc.

https://docs.google.com/spreadsheets/d/1r4dPw1shIEy_W2BkfgPsihinwg-Nah654VlNTn8Gxo0/edit?usp=sharing

If you want you can use the definitions and add them to the MassBank documentation.

Michael

**Comment by michaelwitting** [comment]
Yes, of course you can choose what to use in MassBank. I'm just collecting what people are using, have seen, are expecting.

**Comment by schymane** [comment]
It would also be interesting to provide him the list of all adducts currently in MassBank (in case they aren't in the documentation) so that they are considered in his effort.
